packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,141 @@
+import { css } from "@microsoft/fast-element";
+import { display, focusVisible } from "../../styles";
+import {
+    neutralDividerRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundActiveBehavior,
+    neutralForegroundFocusBehavior,
+    neutralForegroundRestBehavior,
+} from "../../styles/recipes";
+import { heightNumber } from "../../styles/size";
+import { forcedColorsStylesheetBehavior } from "../../styles/match-media-stylesheet-behavior";
+import { SystemColors } from "../../styles/system-colors";
+
+export const AccordionItemStyles = css`
+    ${display("flex")} :host {
+        box-sizing: border-box;
+        font-family: var(--body-font);
+        flex-direction: column;
+        ${
+            /* Font size, weight, and line height are temporary - 
+            replace when adaptive typography is figured out */ ""
+        } font-size: 12px;
+        font-weight: 400;
+        line-height: 18px;
+        border-bottom: calc(var(--outline-width) * 1px) solid var(--neutral-divider-rest);
+    }
+    
+    .region {
+        display: none;
+        padding: calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
+    }
+
+    .heading {
+        display: grid;
+        position: relative;
+        grid-template-columns: auto 1fr auto calc(${heightNumber} * 1px)
+    }
+
+    .button {
+        appearance: none;
+        border: none;
+        background: none;
+        grid-column: 2;
+        grid-row: 1;
+        outline: none;
+        padding: 0 calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
```

Comment:
Would it make sense to hoist this repeated `calc` value to be its own css variable?

Reply:
Yeah its used throughout our components, should be added to the style utilities, I would leave that for another PR though.